pkg/config/seelog/seelog_config.go

```diff
@@ -86,6 +98,14 @@ func (c *Config) ConfigureSyslog(syslogURI string, usetTLS bool) {
 
 }
 
+//ConfigureJMXSpecific Setting JMX specific settings
+func (c *Config) ConfigureJMXSpecific(loggerName string, f string, jsonFormat string, commonFormat string) {
+	c.setValue("loggerName", loggerName)
+	c.setValue("logfile", f)
+	c.setValue("jsonFormat", jsonFormat)
+	c.setValue("commonFormat", commonFormat)
+}
+
```

Comment:
If you create a new config object dedicated for jmx logging you can save this function (or make it more generic if it can be use for both the main & jmx logging config)